tests/js/spec/components/avatar.spec.jsx

```diff
@@ -3,18 +3,20 @@ import {shallow} from 'enzyme';
 import Avatar from 'app/components/avatar';
 
 describe('Avatar', function() {
+  let sandbox;
+
   const USER = {
     id: 1,
     name: 'Jane Doe',
     email: 'janedoe@example.com',
   };
 
   beforeEach(function() {
```

Comment:
it would be nice to have a wrapper or something that reduced a bit of this boilerplate some day